utils/nctl/sh/scenarios/sync_upgrade_test.sh

```diff
@@ -90,6 +93,28 @@ function do_await_network_upgrade() {
     done
 }
 
+function assert_network_upgrade() {
+    local STATUS
+    local COUNT
+    local RUNNING_COUNT
+    local PROTO=${1}
+    log_step "checking that entire network upgraded to $PROTO"
+    CONVERT=$(echo $PROTO | sed 's/_/./g')
```

Comment:
It's more like _converted_ rather than _convert_ (the latter sounds like a verb/method).

Reply:
switched